src/meta/src/rpc/service/ddl_service.rs

```diff
@@ -268,7 +272,7 @@ where
             .catalog_manager
             .drop_sink(sink_id)
             .await
-            .map_err(tonic_err)?;
+            .map_err(meta_error_to_tonic)?;
```

Comment:
Since we already implement `From<MetaError>` for `tonic::Status`, seems unnecessary here?